utils/auth.js

```diff
@@ -0,0 +1,22 @@
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
@osammy already required dotenv before starting the server. Is there another reason why we need it here too?

Reply:
Thanks, that was an oversight. will get rid of it
\